build.sh

```diff
@@ -427,7 +427,7 @@ if [ ${BUILD_BE} -eq 1 ]; then
         cp -r -p ${STARROCKS_THIRDPARTY}/installed/open_jdk/jre/lib/amd64 ${STARROCKS_OUTPUT}/be/lib/jvm/
     fi
     cp -r -p ${STARROCKS_THIRDPARTY}/installed/jindosdk/* ${STARROCKS_OUTPUT}/be/lib/hadoop/hdfs/
-    cp -r -p ${STARROCKS_THIRDPARTY}/installed/broker_thirdparty_jars/* ${STARROCKS_OUTPUT}/be/lib/hadoop/hdfs/
+    cp -r -p ${STARROCKS_THIRDPARTY}/installed/broker_thirdparty_jars/*cos* ${STARROCKS_OUTPUT}/be/lib/hadoop/hdfs/
```

Comment:
not only cos, we also need to other lib in broker_thrid_jars. Since we have remove the oss in build_broker_thirdparty_jars, other jars all should be copied to here i think

Reply:
This implementation retains the original logic conservatively. 
If this is changed as you mentioned , new packages like ks3/cloudfs will be added to the FE/BE lib directory but I also think it is needed.

changed, PTAL